code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -166,6 +201,9 @@
 		if(ALIEN_BODYPART,LARVA_BODYPART) //aliens take double burn //nothing can burn with so much snowflake code around
 			burn *= 2
 
+	if(prob(brute*break_modifier) && ((brute_dam + burn_dam)/max_damage) > 0.3 )
```

Comment:
Why are you accounting burn damage?

Reply:
Burn through the bones (And flattest did it)

Reply:
I guess the argument is that a damaged bone is more likely to be broken regardless if it was scorched or busted?

Reply:
But why

Reply:
I'm pretty sure that if the bone is burned, then the surrounding flesh is already extremely burned.

Reply:
I'd appreciate if this ``break_modifier`` was something easily alterable by config.